x/auth/keeper/deterministic_test.go

```diff
@@ -84,22 +91,42 @@ func (suite *DeterministicTestSuite) runAccountIterations(addr sdk.AccAddress, p
 	}
 }
 
-func (suite *DeterministicTestSuite) TestGRPCQueryAccount() {
-	rapid.Check(suite.T(), func(t *rapid.T) {
+// createAndSetAccount creates a random account and sets to the keeper store.
+func (suite *DeterministicTestSuite) createAndSetAccounts(t *rapid.T, count int) []types.AccountI {
+	accs := make([]types.AccountI, 0, count)
+	accNum := make(map[uint64]bool)
+
+	for i := 0; i < count; i++ {
 		pub := pubkeyGenerator(t).Draw(t, "pubkey")
 		addr := sdk.AccAddress(pub.Address())
-		accNum := rapid.Uint64().Draw(t, "account-number")
+		accNum := rapid.Uint64().Filter(func(u uint64) bool {
+			if accNum[u] {
+				return false
+			} else {
```

Comment:
nit: there is no need for the `else` block..surprised the linter didn't flag this.